app/controllers/land_trusts_controller.rb

```
@@ -0,0 +1,27 @@
+class LandTrustsController < ApplicationController
+  def index
+    land_trusts = LandTrust.all
+    render json: land_trusts, status: 200  
+  end
+
+  def show
+    land_trust = LandTrust.find(params[:id])
+    render json: land_trust, status: 200
+  end
+
+  def destroy
+    land_trust = LandTrust.find(params[:id])
+    land_trust.destroy
```

@harisshafiq making this `land_trust.destroy!` will cause an error to be thrown if the land trust, for whatever reason, can not be deleted. That way, it won't silently fail and return a success message to the user